src/jquery.uls.languagefilter.js

```diff
@@ -46,8 +46,8 @@
 		this.options = $.extend( {}, $.fn.languagefilter.defaults, options );
 		this.$element.addClass( 'languagefilter' );
 		this.resultCount = 0;
-		this.$suggestion = this.$element.parents().find( '#' + this.$element.data( 'suggestion' ) );
-		this.$clear = this.$element.parents().find( '#' + this.$element.data( 'clear' ) );
+		this.$suggestion = this.$element.parents().find( '.' + this.$element.data( 'suggestion' ) );
```

Comment:
Not introduced here, but `parents` is most certainly wrong. It should likely use `parent` or `closest` or even `siblings`.

Reply:
Agree. I will fix it in another PR. fine?

Reply:
Absolutely fine.